sdk/eventhub/event-hubs/src/batchingReceiver.ts

```diff
@@ -116,6 +126,15 @@ export class BatchingReceiver extends EventHubReceiver {
         }
       };
 
+      onAborted = () => {
+        finalAction(timeOver);
```

Comment:
Any reason you chose to call `finalAction()` here?
`finalAction` will resolve the promise returned by `receiveBatch()`. Throwing an error after this point will not reach the user.